frontend/tsconfig.json

```diff
@@ -0,0 +1,38 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "build",
+    "module": "esnext",
+    "target": "ES2017",
+    "lib": [
+      "es6",
+      "dom"
+    ],
+    "sourceMap": true,
+    "allowJs": true,
```

Comment:
Do we really need `allowJs` to be `true` now ?

Reply:
back to false

Reply:
👍